manifests/d/Dege/dgVoodoo2/2.84.1/Dege.dgVoodoo2.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Dege.dgVoodoo2
+PackageVersion: 2.84.1
+InstallerType: zip
+NestedInstallerType: portable
```

```suggestion
NestedInstallerType: portable
ArchiveBinariesDependOnPath: true
```
`./3Dfx` folder in the Zip contains dll files